UserDetails/views.py

```diff
@@ -25,14 +25,14 @@ def get_context_data(self, **kwargs):
         context.update({
             'account_form': RegisterUserForm(),
             'account_detail_form': RegisterUserDetails(),
-            'associationlink_form': RegisterAssociationLinks(),
```

Comment:
`RegisterAssociationLinks` has no usages according to my PyCharm so it should be removed if you're not using it.

Reply:
It is removed? That's what this change is all about...

Reply:
I mean the implementation in `UserDetails/forms.py:37`. According to PyCharm it is not in use anywhere.

Reply:
O, de class zat er nog in. Ah. Zat. zat er in. Zat.

Reply:
:sunglasses: